### DISCUSSION TITLE
New DLL, does it work and how to inject?

### DISCUSSION THREAD

**Discussion by king69666** [YimMenu/YimMenu#3699] [Q&A]
Hey there. So there's a new Nightly build posted a few hours ago before this post. As the originally suggested injector is detected (or at least it gives an error when trying to inject) is there another suggested one to use? And is the new version of the menu even working? I haven't seen any news as of yet about new builds, etc. Just sporadic posts in random threads on UnknownCheats saying Yim works now with no information to back it up.

I assume it's still being worked on, but I'm still curious.

Thanks in advance :D

**Comment by Aholicknight** [comment]
#3690
